ui/src/app/pages/admin/admin-workspace.tsx

```diff
@@ -195,13 +195,13 @@ const FileDetailsTable = (props: FileDetailsProps) => {
 
   return <FlexColumn>
     <DataTable
+        paginator scrollable
```

Comment:
does our one-prop-per-line standard also apply to these?

Reply:
Sure, probably we should have a lint rule for this

Reply:
https://github.com/all-of-us/workbench/pull/6061